internal/cli/alerts/list.go

```diff
@@ -39,6 +39,10 @@ func (opts *ListOpts) initStore() error {
 	return err
 }
 
+var listTemplate = `ID	TYPE_NAME	STATUS{{range .Results}}
```

Comment:
same as other comments
```suggestion
var listTemplate = `ID	TYPE	STATUS{{range .Results}}
```